snuba/query/parsing.py

```diff
@@ -0,0 +1,15 @@
+class ParsingContext:
+    """
+    This class is passed through during the query parsing process
+    to keep any state needed during the process itself (like the
+    alias cache).
+    """
+
+    def __init__(self) -> None:
+        self.__alias_cache = []
```

Comment:
This should have a type annotation since it can't be inferred, probably `MutableSequence[str]`.

Reply:
true